src/lib/ingest.js

```diff
@@ -134,47 +132,44 @@ function logIngestItemsResults(results) {
         // log it as info and not error
         logger.warn('Invalid ingest item', result.error)
       } else {
-        logger.error('Error while ingesting item', result.error)
+        logger.error('Error while ingesting item::', result.error)
```

Comment:
added a double colon, because the downstream errors this surfaces have colons in them, and it was confusing as to how they related.